library/src/main/java/io/realm/RealmBasedRecyclerViewAdapter.java

```diff
@@ -230,6 +232,10 @@ public void onBindHeaderViewHolder(RealmViewHolder holder, int position) {
         }
     }
 
+    public Context getContext() {
```

Comment:
Yes, as explained in https://github.com/thorbenprimke/realm-recyclerview/issues/62 I need the context when extending your class in Kotlin to be able to extract locale based headers, since the super constructor always runs first and `createHeaderFromColumnValue` is called in your constructor.